docs/topics/proc_creating-a-reactive-http-service-with-webflux.adoc

```diff
@@ -0,0 +1,83 @@
+[id='creating-a-reactive-spring-boot-http-service-with-webflux_{context}']
+= Creating a reactive {SpringBoot} HTTP service with WebFlux
+
+Create a basic reactive Hello World HTTP web service using {SpringBoot} and WebFlux.
+
+.Prerequisites
+
+* Java JDK 8 or Java JDK 11 installed
+* Maven installed
+// this configuration might not apply entirely while examples still in tech preview
+* A Maven-based application project configured link:{link-guide-spring-boot}#configuring-your-application-to-use-spring-boot_spring-boot[configured to use Spring Boot]
+
+
+
+.Procedure
+
+. Add `vertx-spring-boot-starter-http` as a dependency in the `pom.xml` file of your project.
+
+.`pom.xml`
+[source,xml,options="nowrap",subs="attributes+"]
+----
+<project>
+...
+  <dependencies>
+  ...
+include::resources/spring-boot/reactive-examples/example-http-service-webflux-pom.xml[tags=http-service-pom-dependencies]
+  ...
+  <dependencies>
+...
+</project>
+----
+
+. Create a main class for your application and define the router and handler methods. 
+
+.`HttpSampleApplication.java`
+[source,java,options="nowrap",subs="attributes+"]
+----
+include::resources/spring-boot/reactive-examples/example-http-service-webflux.java[]
+----
+
+. OPTIONAL: Run and test your application locally 
+
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ cd myApp 
+----
+
+.. Package your application:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ mvn clean package
+----
+
+.. Start your application from the command line:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ java -jar target/vertx-spring-boot-sample-http.jar 
+----
+.. In a new terminal window, issue an HTTP request on the `/hello` endpoint:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ curl localhost:8080/hello
+Hello, World!
+----
+.. Provide a custom name with your request to get a personalized response:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ curl http://localhost:8080/hello?name=John
+Hello, John!
+----
+
+.Additional resources
+
+You can link:{link-guide-spring-boot}#deploying-runtime-application-to-openshift_spring-boot[deploy your application to an OpenShift cluster] using Fabric8 Maven Plugin.
+
+You can also configure your application for link:{link-guide-spring-boot}#deploying-runtime-application-to-standalone-red-hat-enterprise-linux_spring-boot[deployment on standalone Red{nbsp}Hat Enterprise Linux].
+
+
+// add links to: https://spring.io/guides/gs/reactive-rest-service/
```

Comment:
Not finished?

Reply:
Added proper link, caption, and bullet point.